introduction.md

```diff
@@ -0,0 +1 @@
+Good evening. I am called by many names though some are more creative than others. Be subjective - and nice.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.